builders/interact/substrate-api/wallets/subwallet.md

```diff
@@ -0,0 +1,111 @@
+
+## Creating your first Substrate Account {: #creating-your-first-ethereum-account }
+
+First, download and install the [Subwallet extension](https://www.subwallet.app/download.html){target=_blank}. Creating a new account will generate a seed phrase that can derive multiple Ethereum and Substrate accounts. By default, Subwallet will generate a single Ethereum and a single Substrate account, but you can easily derive more from the same seed phrase. Click **Create a new account** to get started. 
+
+![Get started with Subwallet](/images/builders/interact/substrate-api/wallets/subwallet/subwallet-1.png)
+
+On the following screen, you'll be prompted to create a password to secure your new wallet. 
+
+![Create a password for Subwallet](/images/builders/interact/substrate-api/wallets/subwallet/subwallet-2.png)
+
+You'll then be prompted to back up your seed phrase. This is an important step, especially because you have the option to later derive additional accounts from this seed phrase. 
+
+![Back up your seed phrase in Subwallet](/images/builders/interact/substrate-api/wallets/subwallet/subwallet-3.png)
+
+!!! note
+    You should never share your seed phrase (mnemonic) or private key with anyone. This gives them direct access to your funds. This guide is for educational purposes only.
+
+## Importing an Existing Substrate Account {: #importing-an-existing-substrate-account }
+
+Of course, you can import an existing Substrate account into Subwallet. To get started, take the following steps:
+
+1. Press the **All accounts** button at the top
+2. Press the **Import account** icon
+
+![Import account part 1](/images/builders/interact/substrate-api/wallets/subwallet/subwallet-4.png)
+
+On the following screen, select the method by which you would like to import the existing account. If you're importing a Substrate account, you can choose from the seed phrase, Polkadot.js (JSON), or QR code options. 
+
+![Import existing account part 2](/images/builders/interact/substrate-api/wallets/subwallet/subwallet-5.png)
+
+On the following screen you'll be able to provide the relevant seed phrase, JSON file, or QR code, and you can begin using your new account right away. 
+
+## Connecting Subwallet to the Tanssi Dancebox TestNet {: #connecting-Subwallet-to-the-tanssi-dancebox-testnet }
+
+Subwallet comes pre-configured with support for the Tanssi Dancebox Testnet, but it may not be enabled by default. You just need to head to the **Manage networks** page to toggle it on. Remember that the Tanssi Dancebox Testnet itself is not a ContainerChain; it is the Substrate-based parachain that orchestrates and manages the launch of Tanssi ContainerChains. To configure your Subwallet to work with Dancebox, press the **More Options** icon in the upper left corner. Then click **Manage networks**, and take the following steps:
```

Comment:
```suggestion
Subwallet comes pre-configured with support for the Tanssi Dancebox TestNet, but it may not be enabled by default. You just need to head to the **Manage networks** page to toggle it on. Remember that the Tanssi Dancebox TestNet itself is not a ContainerChain; it is the Substrate-based parachain that orchestrates and manages the launch of Tanssi ContainerChains. To configure your Subwallet to work with Dancebox, press the **More Options** icon in the upper left corner. Then click **Manage networks** and take the following steps:
```